CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 - Bump Flipper to v0.131.1
 - Migrate `RegistrationLocationPermissionScreen` to use view effects
 - Bump Lottie to v4.2.2
+- Migrate `RegistrationPinScreen` to use view effects
```

Comment:
There seems to be an extra entry in the changelog

Reply:
oh, that's because `Migrate `RegistrationPinScreen` to use view effects` was wrongly added to the on demo section. I forgot to remove it from the on demo section though. Made that change.